src/API/Management/Blacklists.php

```diff
@@ -15,7 +15,11 @@ public function getAll($aud)
     {
         $response = $this->httpClient->get('/blacklists/tokens?'.http_build_query(['aud' => $aud]));
 
-        return ResponseMediator::getContent($response);
+        if (200 === $response->getStatusCode()) {
+            return ResponseMediator::getContent($response);
+        }
+
```

Comment:
Special exceptions related to this endpoint will be added here.

(Blacklist does not have anything special)